app/jobs/catalogue_check_job.rb

```diff
@@ -0,0 +1,76 @@
+class CatalogueCheckWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'

```

Comment:
In the `Gemfile`, move the capybara and poltergeist gems so they aren't within any group, as we want them available in all environments.

Does this work without the requires on lines 4 and 5?